e2e-tests/monitoring/01-deploy-pmm-server.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    . ../functions
+    set -o errexit
+    set -o xtrace
+
+    helm install monitoring \
+      -n "${NAMESPACE}" \
+      --set platform=kubernetes \
+      https://percona-charts.storage.googleapis.com/pmm-server-2.18.0.tgz
```

Comment:
mb we have to move to more up-to-date version

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/21/commits/757e92c61cb0691b3713cd46d05a7091094ec2e7

although latest version is 2.24.0, in our chart repo latest was 2.23.0